user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures/SLNetClientTest_debugging_cloud_connection.md

```diff
@@ -0,0 +1,57 @@
+---
+uid: SLNetClientTest_debugging_cloud_connection
+---
+
+# Debugging the dataminer.services connection
+
+When there is a problem with the connection to dataminer.services, you can debug this using the *CcaGateway* window in the SLNetClientTest tool. This is both possible when the DataMiner Agent is online or offline (from DataMiner 10.3.8/10.4.0 onwards<!-- RN 36611 -->).
+
+## Accessing the CcaGateway window
+
+When the DataMiner Agent is **online**:
+
+1. [Connect to the DMA using the SLNetClientTest tool](xref:Connecting_to_a_DMA_with_the_SLNetClientTest_tool).
+
+1. Go to *Advanced* > *CcaGateway*.
+
+When the DataMiner Agent is **offline**:
+
+- Go to *Offline Tools* > *CcaGateway (offline)*.
+
+## CcaGateway window UI
```

Comment:
Note that the window has been adjusted over time.
All versions have the Connect & Tunnel Endpoint (name in UI could vary but they are the same fields)
Some older versions had **no or multiple other endpoint fields** that now all have been merged into one: DCP API Endpoint

Reply:
I've added the DataMiner version below the screenshot to deal with possible UI differences in different versions.